docs/gettinginvolved/infrastructure/documentation.rst

```diff
@@ -31,9 +31,11 @@ obsolete releases that are no longer hosted are permanently redirected to the la
      <!-- Marketing Release Documentation -->
      <RoutingRule>
```

Comment:
The automated process only changes the bit of information that changes. There is nothing that automates the creation of the rest of it.